### PR TITLE
functests: remove some code duplication

### PR DESCRIPTION
**Pull request by marcel-apf**
Factor out duplicated code to simple functions.

Signed-off-by: Marcel Apfelbaum <marcel@redhat.com>

### CODE REVIEW

**Comment by dshchedr** [comment]
@marcel-apf can you please add one test_id, anyway you are doing tests cleaning - so I don't want to breed additional pull requests.


https://github.com/openshift-kni/performance-addon-operators/blob/6a22967edabd093051a28d0cf503bfb3f94fe7e0/functests/1_performance/performance.go#L102

this line should be: `It("[test_id:32702]Should set CPU isolcpu's kernel argument managed_irq flag", func() {`

File: functests/0_config/config.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package __performance_config
 import (
 	"context"
 	"fmt"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/nodes"
```
[the remark this answers is not in the document]
Comment (marcel-apf) [reply functests/0_config/config.go:6]: will do

File: functests/1_performance/cpu_management.go [hunk 2]
```diff
@@ -33,18 +33,13 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 	var reservedCPUSet cpuset.CPUSet
 
 	BeforeEach(func() {
-		workerRTNodes, err := nodes.GetByRole(testutils.RoleWorkerCNF)
-		Expect(err).ToNot(HaveOccurred())
+		workerRTNodes, err := nodes.GetCNFNodes()
 		workerRTNodes, err = nodes.MatchingOptionalSelector(workerRTNodes)
```
[the remark this answers is not in the document]
Comment (marcel-apf) [reply functests/1_performance/cpu_management.go:37]: No idea, but I will look into it for the next spin

**Comment by marcel-apf** [comment]
> @marcel-apf can you please add one test_id, anyway you are doing tests cleaning - so I don't want to breed additional pull requests.
> 
> https://github.com/openshift-kni/performance-addon-operators/blob/6a22967edabd093051a28d0cf503bfb3f94fe7e0/functests/1_performance/performance.go#L102
> 
> this line should be: `It("[test_id:32702]Should set CPU isolcpu's kernel argument managed_irq flag", func() {`

Of course, what test_id should I use?

**Comment by dshchedr** [comment]
> > @marcel-apf can you please add one test_id, anyway you are doing tests cleaning - so I don't want to breed additional pull requests.
> > https://github.com/openshift-kni/performance-addon-operators/blob/6a22967edabd093051a28d0cf503bfb3f94fe7e0/functests/1_performance/performance.go#L102
> > 
> > this line should be: `It("[test_id:32702]Should set CPU isolcpu's kernel argument managed_irq flag", func() {`
> 
> Of course, what test_id should I use?

[test_id:32702] 
:)

**Comment by marcel-apf** [comment]
Need to rethink

**Pull request closed without merging**